cli/cmd/project/edit.go

```diff
@@ -12,6 +12,7 @@ func EditCmd(ch *cmdutil.Helper) *cobra.Command {
 	var name, description, prodVersion, prodBranch, subpath, path, provisioner string
 	var public bool
 	var prodTTL int64
+	var slots int
```

Comment:
`slots` and the `prod-slots` flag was removed recently – perhaps this is a broken merge?

Reply:
Most likely, will adjust